source/_includes/head.html

```diff
@@ -31,6 +31,8 @@
   <script src="{{ "/components/bootstrap/dist/js/bootstrap.min.js" | prepend: site.baseurl }}"></script>
   <script src="{{ "/components/patternfly-bootstrap-combobox/js/bootstrap-combobox.js" | prepend: site.baseurl }}"></script>
   <script src="{{ "/components/bootstrap-datepicker/dist/js/bootstrap-datepicker.js" | prepend: site.baseurl }}"></script>
+  <script src="{{ "/components/moment/min/moment.min.js" | prepend: site.baseurl }}"></script>
```

Comment:
Please wrap these script tags with the following conditional so they are only inlcuded on the widgets page (and not on ever page in the site):
```
{% if primary == 'pattern-library' and secondary == 'widgets' # only add for the widgets page %}
...
{% endif %}
```

Reply:
Hi @bleathem , I wanna commit the following code snippets:
```html
  {% if primary == 'pattern-library' and secondary == 'widgets' # only add for the widgets page %}
  <script src="{{ "/components/patternfly-bootstrap-combobox/js/bootstrap-combobox.js" | prepend: site.baseurl }}"></script>
  <script src="{{ "/components/bootstrap-datepicker/dist/js/bootstrap-datepicker.js" | prepend: site.baseurl }}"></script>
  <script src="{{ "/components/moment/min/moment.min.js" | prepend: site.baseurl }}"></script>
  <script src="{{ "/components/eonasdan-bootstrap-datetimepicker/build/js/bootstrap-datetimepicker.min.js" | prepend: site.baseurl }}"></script>
  <script src="{{ "/components/bootstrap-select/dist/js/bootstrap-select.min.js" | prepend: site.baseurl }}"></script>
  <script src="{{ "/components/bootstrap-switch/dist/js/bootstrap-switch.min.js" | prepend: site.baseurl }}"></script>
  <script src="{{ "/components/bootstrap-touchspin/dist/jquery.bootstrap-touchspin.min.js" | prepend: site.baseurl }}"></script>
  <script src="{{ "/components/patternfly-bootstrap-treeview/dist/bootstrap-treeview.min.js" | prepend: site.baseurl }}"></script>
  {% endif %}
```

But bootstrap-datepicker.js is still used in ["Forms and Controls/Datepicker"](http://www.patternfly.org/pattern-library/forms-and-controls/datepicker/).  Shall we remove it from category "Forms and Controls" because there is ["Bootstrap Datepicker"](http://www.patternfly.org/pattern-library/widgets/#bootstrap-datepicker) under category "Widgets"

Reply:
Let's file a new issue to address removing the Bootstrap Datepicker from the forms and controls page